eth/backend.go

```diff
@@ -75,6 +75,7 @@ type Ethereum struct {
 	// Handlers
 	txPool          *core.TxPool
 	blockchain      *core.BlockChain
+	bor             *bor.Bor
```

Comment:
Why do you need bor object here? It will fail in case of `clique` consensus.

Reply:
Also, are we using it somewhere?

Reply:
Removing this
We are using `Ethereum.engine.bor`